documentation/acceptence-tests/group-expense-splitting-tests.md

```diff
@@ -0,0 +1,25 @@
+# Group Expense Splitting Acceptance Tests
+
+## Acceptance test summary
+
+### Create group expense
+
+1. Login as valid user
+2. Navigate to the Group Expenses section
+3. Click Add Group Expense
+4. Fill in the required fields (Title, amount, category)
+5. Fill in a second valid user name
+6. Click save
+7. Ensure the Group expense is added to the list.
+8. Ensure the distribution is correct between users
```

Comment:
add what valid user means, and there are 2 routers for group expense, evenly distributed and user input, you should make this acceptance test into two

Reply:
We only need one since they are part of the same story 

> add what valid user means, and there are 2 routers for group expense, evenly distributed and user input, you should make this acceptance test into two

Reply:
yeah but there are 2 things you are testing in the story 'evenly or unevenly distributed' or if you are gonna keep one, put them as 'evenly distributed group expense'